### ISSUE TITLE
fix: integrate `aria-live` configuration

### ISSUE THREAD

**Issue by krishnanand5** [xability/maidr#334]
We have a two-thronged issue to tackle:
1. The `aria-live` config in settings should be mapped to all applicable components. User should be able to switch between `assertive` and `polite` modes and have it reflected on related UI components dynamically.
2. The scope of assigning `aria-live` attribute to applicable UI components should be analyzed and all related components should be provided `aria-live` support in one go - this will require changes on all UI pages.
